src/protocols/aeternity/composables/aeNames.ts

```diff
@@ -132,6 +133,16 @@ export function useAeNames() {
     return computed(() => externalNamesRegistry.value[nodeNetworkId.value!]?.[address] || '');
   }
 
+  // This function returns the resolved chain name from a name hash
+  async function getChainName(address?: Name) {
```

Comment:
Already existing method `getName` is also returning the chain name, so I thing this method name is not exactly reflecting the difference between those 2.

Reply:
Renamed to `getNameByNameAddress` to avoid confusion